File: 3_zad.go
package main

import (
	"fmt"
	"math"
)

func main() {
	b := 2.5
	b3 := b * b * b

	f := func(x float64) float64 {
		sum := b3 + x*x*x
		return (1 + math.Pow(math.Sin(sum), 2)) / math.Cbrt(sum)
	}

	fmt.Println("Задача А")
	fmt.Println("   x    |     y")
	fmt.Printf("%6.2f  | %8.6f\n", 1.28, f(1.28))
	fmt.Printf("%6.2f  | %8.6f\n", 1.68, f(1.68))
	fmt.Printf("%6.2f  | %8.6f\n", 2.08, f(2.08))
	fmt.Printf("%6.2f  | %8.6f\n", 2.48, f(2.48))
	fmt.Printf("%6.2f  | %8.6f\n", 2.88, f(2.88))
	fmt.Printf("%6.2f  | %8.6f\n", 3.28, f(3.28))

	fmt.Println("\nЗадача Б")
	fmt.Println("   x    |     y")
	fmt.Printf("%5.1f   | %8.6f\n", 1.1, f(1.1))
	fmt.Printf("%5.1f   | %8.6f\n", 2.4, f(2.4))
	fmt.Printf("%5.1f   | %8.6f\n", 3.6, f(3.6))
	fmt.Printf("%5.1f   | %8.6f\n", 1.7, f(1.7))
	fmt.Printf("%5.1f   | %8.6f\n", 3.9, f(3.9))
}

File: 3_zad.ts
const b = 2.5;
const bCubed = b ** 3;

const Func = (x: number) => {
    const sum = bCubed + Math.pow(x, 3);
    return (1 + Math.sin(sum) ** 2) / Math.cbrt(sum);
};

console.log("Задача А");
console.log("   x    |     y");
console.log(`  1.28  | ${Func(1.28).toFixed(6)}`);
console.log(`  1.68  | ${Func(1.68).toFixed(6)}`);
console.log(`  2.08  | ${Func(2.08).toFixed(6)}`);
console.log(`  2.48  | ${Func(2.48).toFixed(6)}`);
console.log(`  2.88  | ${Func(2.88).toFixed(6)}`);
console.log(`  3.28  | ${Func(3.28).toFixed(6)}`);

console.log("\nЗадача Б");
console.log("   x    |     y");
console.log(`  1.1   | ${Func(1.1).toFixed(6)}`);
console.log(`  2.4   | ${Func(2.4).toFixed(6)}`);
console.log(`  3.6   | ${Func(3.6).toFixed(6)}`);
console.log(`  1.7   | ${Func(1.7).toFixed(6)}`);
console.log(`  3.9   | ${Func(3.9).toFixed(6)}`);